src/EditorFeatures/Next/IntelliSense/Completion/Presentation/FilteredRoslynCompletionSet.cs

```diff
@@ -0,0 +1,239 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
I feel like we can do mix-ins with composition.  i.e. just contain the appropritae type and forward along as neessary.  I can help out with trying this out on monday.